users/views.py

```diff
@@ -1,3 +1,69 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
+
+from django.conf  import settings
+from django.views import View
+from django.http  import JsonResponse
+from django.forms import ValidationError
+
+from .models    import User
+from .validator import password_validator
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            user         = data['user']
+            password     = data['password']
+            address      = data['address']
+            phone_number = data['phone_number']
+
+            password_validator(password)
```

Comment:
함수명을 만들때는 동사로 시작하도록 만들어야 합니다.
`함수 == Action`이라고 생각해주세요.
```suggestion
            validate_password(password)
```

Reply:
넵 수정하겠습니다